README.md

```diff
@@ -113,6 +113,49 @@ both served from the `example` folder.
 - `npx react-native start`
 - type `i` or `a` to run on either platform
 
+#### Push Notifications for Android and Firebase services Integration Guide
+
+Push notifications require `react-native-notifications` and Firebase services.
+To integrate with your example app, follow the steps below:
+
+##### 1. Modify MainApplication.java
+
+Add the following imports to `MainApplication.java` file:
+
+```
+import com.google.firebase.FirebaseApp;
+import com.wix.reactnativenotifications.RNNotificationsPackage;
+```
+
+Uncomment the line that initializes Firebase in the `MainApplication.java` file:
+
+```
+FirebaseApp.initializeApp(this);
+```
+
+##### 2. Update package.json
+
+In the example folder, add `react-native-notifications` to the dependencies
+section the the package.json file:
+
+```
+"dependencies": {
+  "react-native-notifications": "^4.3.5",
+  // other dependencies...
+}
+```
+
+example: `yarn add react-native-notifications`
+
+##### 3. Add google-services.json
+
+Please bring in your own `google-services.json` file and add it to the
+appropriate location in your project. The `google-services.json` file is
+required for Firebase services to work correctly.
+
+Make sure to follow the Firebase documentation to obtain the
+`google-services.json` file specific to your Firebase project.
+
```

Comment:
Do we need to set `enableFirebase` and `IS_FIREBASE_ENABLED ` to true? Do we need 2 variables?

Do we need to update `pushNotificationsConfig` on the JS side?

Reply:
my thinking is that if we set `enableFirebase` and `IS_FIREBASE_ENABLED` to true, then the code paths that lead to firebase initialization are going to execute and cause the crash

the difference between these 2 variables is that one is used as a boolean in the gradle files and the other in the java files. So i think that we will still need them both unless there is room to remove conditionals all together like here possibly:
```
if (BuildConfig.IS_FIREBASE_ENABLED) {
  // If you opted-in for Push Notifications, Firebase is initialized.
  // FirebaseApp.initializeApp(this);
}
```
i could see an argument to just get rid of the conditional all together and leave initialization commented until a user is ready to use it

i dont think that we need to touch `pushNotificationsConfig` at this time but im open to ideas and suggestions. I might not be seeing something

Reply:
I think it is fine to remove the conditional like you suggested and leave it commented out as long as there are explicit instructions that lead from the README to this file, and then something like "Uncomment next line to enable firebase".